### ISSUE TITLE
C+ not working in certain cutscenes.

### ISSUE THREAD

**Issue by minisuflower** [XIV-Tools/CustomizePlus#38]
Hello! I noticed that Customize+ is not working in a few cutscenes (no Body Scaling) ie. cutscenes where our character is taking in the light from the lightwardens, and also a scene when entering ratika where the blessed surround us (#2 in into the dark) or Hope's Confluence after the SoS fight. I am unsure if there is more.

**Comment by bestofbatsu** [comment]
Also, Customize+ doesn't work for certain cutscenes in the quest "Return to the Phantom Realm" (quest to unlock Endwalker's Alliance Raid  #2). Maybe depends on the cutscene type?

**Comment by StoiaCode** [comment]
#84, keeping this open for additional info

**Comment by cdoug8** [comment]
since cutscene scaling seems to be my problem to ill add my scaling seems to randomly not work during cutscenes. sometimes it works during non voiced cutscnes and sometimes it dosent work. sometimes it works for voiced cutscenes and sometimes it doesnt. ive even seen it turn off and on during the same cutscene when the camera switches angles.

**Comment by MaelstromFalco** [comment]
So I figured out a workaround, but it unfortunately creates another issue.

I duplicated my character's C+ profile and left the character name blank and enabled it. That allowed the duplicate scale to show up for my character in the cutscenes that didn't include the original C+ scale.

The problem is that the scale also applies to other untargetable no name npcs. I also tried turning on the "Only Owned" checkbox on the duplicate profile, but that turns off the scale for both the player character in those certain cutscenes and the untargetable no name npcs. It's as if the player in those cutscenes are not considered player owned.

Apologies for any confusion.

**Comment by StoiaCode** [comment]
You are right, they are not. The "Player" character in those cutscenes are random nameles Characters that look like you, but are in no actual way connected to you, at least not via name etc.

**Comment by MaelstromFalco** [comment]
What I find strange is that Penumbra profiles and Palette+ seems to work on these PC lookalike npc actors in those cutscenes, but not C+.

**Comment by StoiaCode** [comment]
They use a different detection algorithm. The one in C+ is rather "primitive" in regards, that it simply scans all objects and checks the name field. Both others have more intricate ones that go out of their way to make sure they catch your character in those cases.

**Comment by MaelstromFalco** [comment]
Understood. Thanks for responding. Hopefully you can get this fixed one day.
